marshal.go

```diff
@@ -8,6 +8,11 @@ import (
 	"time"
 )
 
+type FuncHandle struct {
```

Comment:
i think, it's better to rename:
```
FuncHandle -> Func
FuncHandle.Func -> Name
FuncHandle.Data -> Args
```

it will make interface more obvious:
```go
query, err := clickhouse.BuildInsert("clicks",
    clickhouse.Columns{"name", "date", "sourceip"},
    clickhouse.Row{"Test name", "2016-01-01 21:01:01", clickhouse.Func{"IPv4StringToNum", "192.0.2.192"}}
)
```

i also believe Func should be moved to query.go